docs/reference/method/MongoDBModelIndexInfo-isTtl.txt

```diff
@@ -28,6 +28,23 @@ Return Values
 
 A boolean indicating whether the index is a TTL index.
 
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $info = new IndexInfo([
+       'expireAfterSeconds' => 100,
+   ]);
+
+   var_dump($info->isTtl());
```

Comment:
Do we really use ``isTtl`` and not ``isTTL``?

Reply:
[Yes](https://github.com/mongodb/mongo-php-library/blob/master/src/Model/IndexInfo.php#L130), we do!